x-pack/plugins/cases/server/routes/api/sub_case/find_sub_cases.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import Boom from '@hapi/boom';
+
+import { pipe } from 'fp-ts/lib/pipeable';
+import { fold } from 'fp-ts/lib/Either';
+import { identity } from 'fp-ts/lib/function';
+
+import { SubCasesFindRequestRt, throwErrors } from '../../../../common/api';
+import { RouteDeps } from '../types';
+import { escapeHatch, wrapError } from '../utils';
+import { SUB_CASES_URL } from '../../../../common/constants';
+
+export function initFindSubCasesApi({ router, logger }: RouteDeps) {
+  router.get(
+    {
+      path: `${SUB_CASES_URL}/_find`,
+      validate: {
+        params: schema.object({
+          case_id: schema.string(),
+        }),
+        query: escapeHatch,
+      },
+    },
+    async (context, request, response) => {
+      try {
+        const queryParams = pipe(
+          SubCasesFindRequestRt.decode(request.query),
+          fold(throwErrors(Boom.badRequest), identity)
+        );
+
+        const client = await context.cases.getCasesClient();
+        return response.ok({
+          body: await client.subCases.find({
+            caseID: request.params.case_id,
```

Comment:
Maybe something for your follow up PR but I know there are a lot of places with `caseId` (lowercase d). Perhaps we should be consistent with how we camelcase ID